ksqldb-engine/src/main/java/io/confluent/ksql/utilization/PersistentQuerySaturationMetrics.java

```diff
@@ -55,6 +54,7 @@
   private static final String STREAMS_THREAD_METRICS_GROUP = "stream-thread-metrics";
   private static final String THREAD_ID = "thread-id";
   private static final String QUERY_ID = "query-id";
+  private static Map<String, String> customTags = new HashMap<>();
```

Comment:
this can be final

Reply:
I'm not sure it can, we set it to have the passed in tags in `setTags` and `configureShared`